lib/browser/api/browser-window.js

```diff
@@ -162,6 +162,9 @@ Object.assign(BrowserWindow.prototype, {
   inspectServiceWorker () {
     return this.webContents.inspectServiceWorker()
   },
+  showCertificate () {
```

Comment:
You are right. Removed in https://github.com/brave/muon/commit/4deb4b2bfa515889c69bd9e44aa7d80c858dad73